_posts/en/newsletters/2021-11-24-newsletter.md

```diff
@@ -0,0 +1,82 @@
+---
+title: 'Bitcoin Optech Newsletter #176'
+permalink: /en/newsletters/2021/11/24/
+name: 2021-11-24-newsletter
+slug: 2021-11-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter links to a discussion about how to allow LN users
+to choose between higher fees and higher payment reliability and
+includes our regular sections with popular questions and answers from
+the Bitcoin StackExchange, announcements of new releases and release
```

Comment:
Below, Stack Exchange is written as two words, here it is a single word. "Stack Exchange" as two words is also the title given by the company that is running the network.

```suggestion
the Bitcoin Stack Exchange, announcements of new releases and release
```

Reply:
From a cursory search, we have used both in the past. I would be happy to volunteer to harmonize the spelling across all of our posts if such a PR were welcome.

Reply:
We previously agreed on two words going forward, this was just a typo by me.  I don't personally care about changing old posts but I also don't have an objection to it if you want to put in the work.

Reply:
Ah okay, I wasn't sure what we had ended up deciding last time. :+1:

Reply:
It was luckily easy: https://github.com/bitcoinops/bitcoinops.github.io/pull/690

Reply:
Thanks @Xekyo !